src/Analyzer/Utilities/RuleFinder.cs

```diff
@@ -28,7 +28,6 @@ public RuleFinder()
             new[]
             {
                 Assembly.Load(typeof(FlagsEnumZeroMember).Assembly.GetName()),
-                Assembly.Load(typeof(FlagsEnumZeroMemberBase).Assembly.GetName())
```

Comment:
This is an abstract class that now has a type parameter, which made compilation fail.
Removing this seems to not create problems since `FlagsEnumZeroMember` extends it and including that needs this one.

Reply:
I will believe you! :D